wgsl/index.bs

```diff
@@ -1277,17 +1277,20 @@ The following table lists the correspondence between WGSL texel formats and
 
 ### Storage Texture Types ### {#texture-storage}
 
-A <dfn noexport>read-only storage texture</dfn> supports reading a single texel without the use of a sampler,
-with automatic conversion of the stored texel value to a usable shader value. A <dfn noexport>write-only storage
-texture</dfn> supports writing a single texel, with automatic conversion
+A <dfn noexport>readable storage texture</dfn> has a `read` or `read_write`
```

Comment:
I don't think we can have `read_write` storage textures can we? I thought that was a limitation of one of the underlying APIs.

Reply:
Ah, in which case I'll drop the PR.